ui/packages/consul-ui/app/services/client/http.js

```diff
@@ -26,8 +26,9 @@ export const restartWhenAvailable = function(client) {
     throw e;
   };
 };
-const stringifyQueryParams = createQueryParams(encodeURIComponent);
-const parseURL = createURL(encodeURIComponent, stringifyQueryParams);
+const QueryParams = {
+  stringify: createQueryParams(encodeURIComponent),
+};
```

Comment:
I made this look like `JSON.stringify` as I vaguely remember doing the same elsewhere in the app. At somepoint we could probably move this to a separate import and use this interface in both places.